src/frontend/citizen-portal/src/app/components/dispute-ticket-stepper/dispute-ticket-stepper.component.ts

```diff
@@ -184,7 +184,8 @@ export class DisputeTicketStepperComponent implements OnInit, AfterViewInit {
   public onDLProvinceChange(province) {
     setTimeout(() => {
       if (province == "BC") {
-        this.form.get("drivers_licence_number").setValidators([Validators.maxLength(9)])
+        this.form.get("drivers_licence_number").setValidators([Validators.maxLength(9)]);
+        this.form.get("drivers_licence_number").addValidators([Validators.minLength(7)]);
       } else {
```

Comment:
Not necessary but can be setValidators([Validators.minLength(7), Validators.maxLength(9)])